kernelci/dashboard
# feature: make hardware listing timestamp based

# Description
Due to our limitation with URLs we can't pass all the data there, so we went for a timestamp based approach so in ten years we can display something similar that the user saw today

## How to test
Just go to hardware listing, enter any hardware details page, it should be sending the timestamps to the backend
